src/__tests__/config/attachmentConfig.test.ts

```
@@ -0,0 +1,415 @@
+				expect(errorMessages.discordUploadFailed).toBe('📤 Failed to upload attachment to Discord.');
+				expect(errorMessages.attachmentProcessingFailed).toBe('🔄 Attachment processing failed, please try again.');
+			});
+
+			it('should include emojis for user-friendly experience', () => {
+				const errorMessages = DISCORD_ATTACHMENT_CONFIG.errorMessages;
+				
+				// Check that all error messages contain emojis
+				Object.values(errorMessages).forEach(message => {
+					expect(message).toMatch(/[⚠️📏📎🔄❌⬇️⏱️🔑📤]/);
+				});
+			});
```

Biome (2.1.2)

[error] A character class cannot match a character and a combining character.

A character and a combining character forms a new character. Replace the character class with an alternation.

(lint/suspicious/noMisleadingCharacterClass)